src/api/v1/community_businesses/visit_activities.ts

```diff
@@ -0,0 +1,166 @@
+}
+interface PutRequest extends Hapi.Request {
+  payload: Partial<VisitActivity>;
+}
+
+export default [
+  {
+    method: 'GET',
+    path: '/community-businesses/me/visit-activities',
+    options: {
+      description: 'Retrieve all visit activities for a community business',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['visit_activities-own:read'],
+        },
+      },
+      pre: [
+        { method: getCommunityBusiness, assign: 'communityBusiness' },
+      ],
+      validate: {
+        query: visitActivitiesGetQuery,
+      },
+      response: { schema: response },
+    },
+    handler: async (request: GetRequest, h: Hapi.ResponseToolkit) => {
+      const { knex, pre: { communityBusiness }, query: { day: _day = null } } = request;
+      const day = _day === 'today'
+        ? <Day> moment().format('dddd').toLowerCase()
+        : _day;
+
+      return CommunityBusinesses.getVisitActivities(knex, communityBusiness, day);
+    },
+  },
+  {
+    method: 'POST',
+    path: '/community-businesses/me/visit-activities',
+    options: {
+      description: 'Retrieve all visit activities for a community business',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['visit_activities-own:write'],
+        },
+      },
+      pre: [
+        { method: getCommunityBusiness, assign: 'communityBusiness' },
+      ],
+      validate: {
+        payload: visitActivitiesPostPayload,
+      },
+      response: { schema: response },
+    },
+    handler: async (request: PostRequest, h: Hapi.ResponseToolkit) => {
+      const { knex, payload: visitActivity, pre: { communityBusiness } } = request;
+      return CommunityBusinesses.addVisitActivity(knex, visitActivity, communityBusiness);
+    },
+  },
+  {
+    method: 'PUT',
+    path: '/community-businesses/me/visit-activities/{visitActivityId}',
+    options: {
+      description: 'Retrieve all visit activities for a community business',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['visit_activities-own:write'],
+        },
+      },
+      pre: [
+        { method: getCommunityBusiness, assign: 'communityBusiness' },
+      ],
+      validate: {
+        payload: visitActivitiesPutPayload,
+        params: {
+          visitActivityId: id,
+        },
+      },
+      response: { schema: response },
+    },
+    handler: async (request: PutRequest, h: Hapi.ResponseToolkit) => {
+      const { knex, payload } = request;
+
+      return CommunityBusinesses.updateVisitActivity(knex, payload);
+    },
+  },
+  {
+    method: 'DELETE',
+    path: '/community-businesses/me/visit-activities/{visitActivityId}',
+    options: {
+      description: 'Retrieve all visit activities for a community business',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['visit_activities-own:delete'],
+        },
+      },
+      validate: {
+        payload: Joi.any().valid(null),
```

Comment:
Yes,  but `payload: Joi.any().forbidden()` means that they key payload is forbidden rather than just it's value, so this fails any request sent to this endpoint